core/src/main/java/org/lflang/generator/LFGeneratorContext.java

```diff
@@ -35,7 +35,7 @@ enum BuildParm {
     RUNTIME_VERSION("Specify the version of the runtime library used for compiling LF programs."),
     SCHEDULER("Specify the runtime scheduler (if supported)."),
     TARGET_COMPILER("Target compiler to invoke."),
-    THREADING("Specify whether the runtime should use multi-threading (true/false)."),
+    SINGLE_THREADED("Specify whether the runtime should use multi-threading (true/false)."),
```

Comment:
This makes it seem like you have to pass "true" or "false" as in `--single-threaded true`. But isn't the arity now zero?

Reply:
That's true, I'd left it because the `SINGLE_THREADED` build param is still boolean (as opposed to the CLI option that now has zero-arity), but I'll remove the "true/false" part so it's consistent with the other boolean build params.